awscli/examples/kms/get-key-policy.rst

```diff
@@ -0,0 +1,20 @@
+**To copy a key policy from one CMK to another CMK**
+
+The following ``get-key-policy`` example gets the key policy from one CMK and saves it in a text file. Then, it replaces the policy of a different CMK using the text file as the policy input.
+Because the ``--policy`` parameter of ``put-key-policy`` requires a string, you must use the ``--output text`` option to return the output as a simple text string.
```

Comment:
Please delete "simple text". It's just a string.

Reply:
My intent was to distinguish between **--output text** and **--output json**, which is also, technically a string.

Reply:
I'll change it to "...return the output as a simple string instead of JSON."

Reply:
How about "return the output as a text string instead of JSON."

Reply:
Ok.  Will do.

Reply:
I changed it as you requested - however, JSON is also a text string, so I'm not sure that's the best way to explain it.